### DISCUSSION TITLE
Disable query output

### DISCUSSION THREAD

**Discussion by YoshieraHuang** [SeaQL/sea-orm#291] [Q&A] [answered]
Hi guys, could you tell me how to disable SQL logging to the console, which like:
```
2021-11-04T12:37:45.907683Z  INFO sqlx::query: /* SQLx ping */; rows: 0, elapsed: 193.771µs  
2021-11-04T12:37:45.908918Z  INFO sqlx::query: SELECT "products"."id", "products"."name", "products"."price" …; rows: 0, elapsed: 1.153ms
 
SELECT
  "products"."id",
  "products"."name",
  "products"."price"
FROM
  "products"
ORDER BY
  "products"."price" DESC
```
Thank you!

**Comment by AngelOnFira** [comment, marked as answer]
It depends on your logger, but this is how I do it:
```rust
Builder::new()
    .filter_level(log::LevelFilter::Info)
    .filter_module("sqlx::query", log::LevelFilter::Off)
    .init();
```

(I'm using the log crate with env_logger)

**Comment by tyt2y3** [comment]
https://docs.rs/sea-orm/0.3.2/src/sea_orm/database/mod.rs.html#40

Apparently I forgot to expose a public method for it. Will do that soon.

Thanks @AngelOnFira ! His suggestion works too and is actually more flexible as it allows the consumer to make the decision instead of the producer.

**Comment by tyt2y3** [comment]
Actually, we later added a `sqlx_logging` field in `ConnectOptions` https://docs.rs/sea-orm/latest/sea_orm/struct.ConnectOptions.html#method.sqlx_logging

**Reply by wojciechbator** [reply]
Would you guys consider to disable logging on a query level rather than on a whole connection? An example is a healthcheck which would log simple query like `SELECT 1` - I'd like to turn off logging for this specific query, rather than on a whole connection, that may handle other requests too.

**Reply by tyt2y3** [reply]
There is a `ping` method in SQLx. May be we can add one (but with our own implementation) too to bypass the statement logging.
https://docs.rs/sqlx/latest/sqlx/trait.Connection.html#tymethod.ping

**Reply by wojciechbator** [reply]
yeah, exposing ping would be very helpful. Fingers crossed it can enter sea-orm soon :)
